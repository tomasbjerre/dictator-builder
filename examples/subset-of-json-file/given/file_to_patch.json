{
  "a": "b",
  "c": "d",
  "e": "f",
  "scipts": {
    "test": "ls"
  }
}
