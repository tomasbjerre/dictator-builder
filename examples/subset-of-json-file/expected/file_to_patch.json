{"a":"1","c":"d","e":"f","g":"2"}